app/views/recipe/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="mb-3 text-center">Recipes</h2>
+      <%= button_to 'Add Recipe', new_recipe_path, method: :get, class: 'btn btn-primary btn-sm' %>
+      <ul class="list-group">
+        <% if @recipes.any? %>
+        <% @recipes.each do |recipe| %>
+          <li class="list-group-item d-flex justify-content-between align-items-center">
+            <div>
+             <h3><%= link_to recipe.name, recipe_path(recipe) %></h3>
+            <% if can? :manage, recipe %>
+  <%= button_to "Remove", recipe, method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-danger btn-sm" %>
+<% end %>
+            </div>
+            <p class="text-muted"><%= recipe.description %></p>
+
+          </li>
```

Comment:
- [ **OPTIONAL** ] Good job as your design follow the **[provided wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_index.png?token=GHSAT0AAAAAAB56UIU3ZZPWCT3YG3NBDG52ZJRMAJQ)**. However, I think it would be better if you improve again your design to look more professional as now when the text description is long text are overlapping the recipe box and text-alignment are set to right by default, such as the image below:
![image](https://github.com/RileyManda/recipe-app/assets/83042742/a8412a96-1b1a-40f3-af39-9fd99f671d64)